infra/cifuzz/external-actions/build_fuzzers/action.yml

```diff
@@ -0,0 +1,40 @@
+# action.yml
```

Comment:
This is good for now. But I was considering merging the two actions before we make this public.
pros/cons of 1 action:
Pros:
1. Simpler set up.
2. Saves a few seconds building another docker image.

Cons:
1. Maintenance cost of having a different system for OSS-Fuzz
2. Log viewing becomes harder, you won't be able to view just run logs, they will be combined with build logs.

Reply:
Hmm, that's a good point to consider. It does seem to me that these 2 steps are different enough that they should be decoupled as two separate steps though. We may want to keep this for flexibility in the future.